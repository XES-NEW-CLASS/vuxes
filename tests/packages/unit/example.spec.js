import { shallowMount } from '@vue/test-utils'
import { findAllDom } from '../../jestUtils/dom'

import router from '@/router'
import App from '@/App.vue'

describe('测试 App.vue', () => {
  it('期望app.vue文件中存在app元素', () => {
    const wrapper = shallowMount(App, {
      router
    })
    expect(findAllDom(wrapper, '#app').length).toBe(1)
    expect(wrapper.element).toMatchSnapshot()
  })
})
