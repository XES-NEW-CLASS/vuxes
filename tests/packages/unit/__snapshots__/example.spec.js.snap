// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`测试 App.vue 期望app.vue文件中存在app元素 1`] = `
<div
  id="app"
>
  <div>
    <p>
      学而思新课堂辅导端 UI 组件库
    </p>
     
    <router-view-stub
      name="default"
    />
  </div>
</div>
`;
